apps/calendar/js/utils/animated_scroll.js

```diff
@@ -0,0 +1,34 @@
+define(function(require, exports, module) {
+'use strict';
+
+module.exports = function animatedScroll(opts) {
+  var {container, content, scrollTop} = opts;
+
+  var maxScroll = container.scrollHeight - container.clientHeight;
+  scrollTop = Math.min(Math.max(scrollTop, 0), maxScroll);
+
+  var diff = container.scrollTop - scrollTop;
+  var seconds = Math.abs(diff) / 500;
```

Comment:
right now this is only really used on the month/week views.. so I think we should keep it simple and improve it later if needed.